packages/fetchai/skills/coin_price/handlers.py

```diff
@@ -0,0 +1,192 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2019 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+"""This package contains handlers for the coin_price skill."""
+
+import json
+from typing import cast
+
+from aea.mail.base import EnvelopeContext
+from aea.protocols.base import Message
+from aea.skills.base import Handler
+
+from packages.fetchai.protocols.http.message import HttpMessage
+from packages.fetchai.skills.coin_price.dialogues import HttpDialogue, HttpDialogues
+
+
+# Skill can be run with or without http_server
```

Comment:
I'd prefer this was explicitly configured on the skill config and then the configuration was cross-checked with the values here to avoid surprises.